<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notification</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	
	background-color: #272b30;
}

QPushButton {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #52575c, stop:1 #3a3e41);

   border: none;
	color:#ffffff;
	padding:5px;
	border-radius : 5px;
   border: 1px solid #272b30;
}

QPushButton:hover {
	color: #3a3f44;
  background-color: #e6e6e6;
}


 *[warning=&quot;true&quot;]
{
	color:lightgrey;
	font-size:8pt;
}


 *[titleLabel=&quot;true&quot;]
{
	color:lightgrey;
	font-size:12pt;
	margin-right: 15px;
	font-family: Verdana;
	padding-bottom:5px;
}

 *[windowControlBtn=&quot;true&quot;]
{
	background-color: rgba(0,0,0,0);
	width: 18px;
	height: 18px;
	
}</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
      <item>
       <widget class="QLabel" name="labelImage">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelEvent">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: rgb(255, 255, 255);
    font-weight: bold;
    font-family: Verdana; 
    font-size:13pt;
}


</string>
        </property>
        <property name="text">
         <string>Lorem ipsum dolor sit amet, consetetur sadipscing elitr</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
	background-color: #2e3236;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="_3">
      <item>
       <widget class="QLabel" name="labelIcon">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>ICO</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelTime">
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: lightgrey;
	font-size: 13px;
}</string>
        </property>
        <property name="text">
         <string>19:24</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
